templates/cassiopeia/scss/blocks/_global.scss

```diff
@@ -89,3 +89,13 @@ meter {
     color: var(--cassiopeia-color-link);
   }
 }
+
+.awesomplete {
+  input {
+    @include border-right-radius(0);
+  }
+}
```

Comment:
@N6REJ What has awesomeplete to do with this PR?

Reply:
pretty sure I didn't put it there

Reply:
regarding the scroll it wasn't that way before.  I believe it's due to the changes you made.  I spent quite a bit of time getting the block uniform.